### PR TITLE
Adding Providers

### PR DESCRIPTION
**Pull request by nielsole**

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -32,6 +32,7 @@
 [[constraint]]
   branch = "master"
   name = "github.com/nextgenhealthcare/cloudhealth-sdk-go"
+  source = "github.com/flix-tech/cloudhealth-sdk-go"
```
Comment (brandonstevens) [inline Gopkg.toml]: Looks like there are a few artifacts like this across the gopkg files that are leftover from local development. They need to be cleaned up before merge. If it helps, I just merged in related SDK changes so you can vendor from the master branch.

**Comment by brandonstevens** [comment]
Looking forward to getting these changes merged in once the vendoring feedback is addressed. Thanks for the contribution!

**Comment by nielsole** [comment]
Which vendoring feedback are you referring to? :)

**Comment by matikij** [comment]
Hmm, not sure which "vendoring feedback" are you referring to. Is it about messing with `Gopkg.lock` and/or `vendor/github.com/ulikunitz/xz/cmd/gxz/*.go` ?

**Review by brandonstevens — APPROVED** [review]
Sorry for the delay. I made the vendoring changes myself. Thanks again for the contribution.

**Pull request merged**